app/src/main/java/nl/ictrek/ananas/Contact.kt

```diff
@@ -0,0 +1,23 @@
+package nl.ictrek.ananas
+
+import org.apache.commons.lang3.text.WordUtils
+
+/**
+ * Created by wouter on 18-4-17.
+
+ * Contact class that contains details about a contact
+ *
+ * @param username The Matrix.org username for this contact
+ * @param email The email for this contact
+ * @param phoneNumber The phone number for this contact
+ * @param fullName The full name for this user
+ */
+class Contact(val username: String, val email: String? = null, val phoneNumber: String? = null, val fullName: String? = null) {
+    val bestName: String
+        get() {
+            if (fullName != null)
```

Comment:
Prefer using braces `{ }` around the body of if-else statements, in case more lines need to be added in the future.